src/linker/Linker.Dataflow/ValueNode.cs

```diff
@@ -905,10 +901,10 @@ protected override string NodeToString ()
 	/// </summary>
 	class MethodReturnValue : LeafValueWithDynamicallyAccessedMemberNode
 	{
-		public MethodReturnValue (MethodReturnType methodReturnType, DynamicallyAccessedMemberTypes dynamicallyAccessedMemberTypes)
+		public MethodReturnValue (MethodBodyScanner scanner, MethodReturnType methodReturnType, DynamicallyAccessedMemberTypes dynamicallyAccessedMemberTypes)
```

Comment:
Last one to move into the caller

Reply:
Yeah, working on it as it can be simplified differently and I didn't want to mix it with the previous ones

Reply:
Done